setup.sh

```diff
@@ -21,7 +22,11 @@ if [ ! -e $env ]; then
 fi
 
 echo "=== Install Python packages into $env..."
-$env/bin/pip install sqlalchemy alembic pyyaml watchdog || exit 1
+$env/bin/pip install -r $codalabdir/requirements.txt || exit 1
+
+# Use sckoo@stanford.edu's patched argcomplete, until pull request has been accepted
+# https://github.com/kislyuk/argcomplete/pull/118
+$env/bin/pip install git+https://github.com/kashizui/argcomplete.git@allow-overridden-subparsersaction
```

Comment:
This point is not specific to this CL, but I noticed it just now.  When I ran ./setup.sh in the codalab-cli, the webserver didn't work, because it was using a different venv.  I guess we have an implicit assumption that the codalab-cli and codalab repos share the same venv?